src/common/SIRF.py

```diff
@@ -155,6 +155,31 @@ def get_uniform_copy(self, value=1.0):
         y.fill(value)
         return y
 
+    def fill(self, value=0):
+        return
+
+    def allocate(self, value=0, **kwargs):
+        """Allocates a copy of self and fills with values
+
```

Comment:
Please make the docstring more complete: some info on `random` (referring to  `numpy.random.random_sample` and hence between 0 and 1),  `random_int`, `max_value`. I'm assuming this signature is determined by CIL? @paskino ?

Reply:
@evgueni-ovtchinnikov can you address this one and then merge?

Reply:
The signature is the same as [CIL](https://github.com/TomographicImaging/CIL/blob/a754e68b3fe9753009c52a2a217ddd381a6c9a62/Wrappers/Python/cil/framework/framework.py#L470-L477C12)